scripts/remove_boilerplate.py

```diff
@@ -191,11 +204,36 @@ def parse_arguments():
     return args
 
 
+def good_token(token):
+    # A token is good if:
+    # * It is on the whitelist
+    # * It is a word ending in ...
+    # * It does not have 3 consecutive punctuation marks
+    global whitelist
```

Comment:
You don't need this if you don't want to modify `whitelist`.

Reply:
Ok.